base/auspice_export.py

```diff
@@ -39,6 +39,23 @@ def process_freqs(freq):
     else:
         process.log.notify("Cannot export frequencies - pivots do not exist")
 
+def export_tip_frequency_json(process, prefix, indent):
+    def process_freqs(freq): # 6dp here, 4 above
+        return [round(x,6) for x in freq]
```

Comment:
+1 for rounding to 6 instead of 4. Instead of defining a replacement function inside a function, we should just define a constant at the top of the module and use the same value and `process_freqs` in all exports for consistency of code and data. Maybe `FREQUENCY_PRECISION = 6` or `EXPORT_PRECISION`, etc.

Reply:
8f4a9ef12c9c548f6696dc888b851cf59879f575